components/BlurhashImage.vue

```diff
@@ -59,13 +60,63 @@ export default Vue.extend({
     };
   },
   mounted(): void {
-    if (this.item.ImageTags && this.item.ImageTags.Primary) {
-      const card = this.$refs.card as HTMLElement;
-      this.image = this.getImageUrlForElement(
-        this.item,
-        ImageType.Primary,
-        card
-      );
+    const img = this.$refs.img as HTMLElement;
+    this.image = this.getImageUrlForElement(this.item, this.type, img);
+  },
+  methods: {
+    isValidTag(): boolean {
+      if (
+        (this.item?.ImageTags &&
+          Object.prototype.hasOwnProperty.call(
+            this.item?.ImageTags,
+            this.type
+          )) ||
+        (this.type === ImageType.Backdrop &&
+          this.item?.BackdropImageTags &&
+          this.item?.BackdropImageTags.length > 0)
+      ) {
+        return true;
+      } else {
+        console.error('Provided tag does not exist in the item');
```

Comment:
I know we shouldn't use console, but throw halts the client altogether. I don't think silencing them is a good idea either, so what do you think?

Reply:
I've let a few of console.error where it is critical, but @MrTimscampi wanted to look more into it for modal stacktraces. So I think for now console errors are good. You can disable the linter going crazy with `// eslint-disable-next-line no-console` on the line before.